backend/open_webui/utils/filter.py

```
@@ -1,24 +1,24 @@
+    load_function_module_by_id,
+    get_function_module_from_cache,
+)
 from open_webui.models.functions import Functions
 from open_webui.env import SRC_LOG_LEVELS
 
 log = logging.getLogger(__name__)
 log.setLevel(SRC_LOG_LEVELS["MAIN"])
 
 
-def get_function_module(request, function_id):
+def get_function_module(request, function_id, load_from_db=True):
```

Ruff (0.11.9)

Missing return type annotation for public function `get_function_module`

(ANN201)